packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -533,6 +533,17 @@ export interface IPendingRuntimeState {
 	 * Time at which session expiry timer started.
 	 */
 	sessionExpiryTimerStarted?: number | undefined;
+	/**
+	 * The snapshots of virtualized dataStores that were downloaded
+	 */
+	downloadedSnapshotTrees?: IDownloadedSnapshotTrees;
```

Comment:
i would like to see a design document for this work, and specifically i would like to see an evaluation of handling blobs like these at the load/container layer, rather than in the runtime

Reply:
Added a design doc to this task [AB#7243](https://dev.azure.com/fluidframework/internal/_workitems/edit/7243), lets discuss there.